creator-node/src/snapbackSM.js

```diff
@@ -449,71 +449,67 @@ class SnapbackSM {
   }
 
   /**
-   * Track an ongoing sync operation for a given secondaryUrl and user wallet
-   *  - Will re-enqueue sync if it fails or is still behind after retries or max duration
+   * Monitor an ongoing sync operation for a given secondaryUrl and user wallet
+   * Return boolean indicating if an additional sync is required
+   *
+   * Polls secondary for MaxSyncMonitoringDurationInMs
    */
-  async monitorSecondarySync (userWallet, primaryClockValue, secondaryUrl, syncType) {
-    const startTime = Date.now()
+  async additionalSyncIsRequired (userWallet, primaryClockValue, secondaryUrl, syncType) {
```

Comment:
maybe hot take but i liked the old function name better lol

Reply:
agreed. just figured it's not worth refactoring again

Reply:
nah i changed it back - if you both liked it, all good

Reply:
kk after on-call discussion will leave as is